shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/uimodel/QuestionUiModel.kt

```diff
@@ -32,3 +35,17 @@ fun Question.toQuestionUiModel(index: Int, totalStep: Int): QuestionUiModel = Qu
     question = this,
     step = "${index + 1}/${totalStep}"
 )
+
+fun List<QuestionUiModel>.mapToQuestionSubmissions(): List<QuestionSubmission> {
+    return this.map { question ->
+        QuestionSubmission(
+            id = question.id,
+            answers = question.answers.mapToAnswerSubmissions()
```

Comment:
Should we map `question.answers` or `question.userInputs` here?

Reply:
I made a mistake; it should be `question.userInputs.toAnswerSubmission()`. Thank you for pointing out.

Reply:
solved in 0318aa5